config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I wonder if we need to get in the habit of removing the old defaults? Doing `config.load_defaults 6.0` I think only loads this file and does nothing with the others. Nothing to change with the PR, just thinking out loud.